stateoftheuniverse/widgets/utils.py

```diff
@@ -0,0 +1,42 @@
+"""
+Utility function for widgets.
+"""
+
+# -----------------------------------------------------------------------------
+# IMPORTS
+# -----------------------------------------------------------------------------
+
+from typing import Callable
+
+
+# -----------------------------------------------------------------------------
+# FUNCTION DEFINITIONS
+# -----------------------------------------------------------------------------
+
+def stringdecorator(function: Callable) -> Callable:
+    """
+    Decorator function which can be used to easily add dashed lines,
+    name, etc. to the `get_string()` method of a widget.
+
```

Comment:
I'm not really sure how to use this. Could you add an example in the docstring pretty pretty please? :)

Reply:
Done!